Makefile

```diff
@@ -11,6 +11,7 @@ KIT ?= bottlerocket-core-kit
 UNAME_ARCH = $(shell uname -m)
 ARCH ?= $(UNAME_ARCH)
 VENDOR ?= bottlerocket
+UPSTREAM_SOURCE_FALLBACK="false"
```

Comment:
We can already set `BUILDSYS_UPSTREAM_SOURCE_FALLBACK` https://github.com/bottlerocket-os/twoliter/blob/2ae29fa73ffe5f1bb7cbcb2799a1e169b3478314/twoliter/embedded/Makefile.toml#L91

Reply:
So, running `BUILDSYS_UPSTREAM_SOURCE_FALLBACK=true make` doesn't pass the setting into twoliter. To pass options to twoliter you would need to do something like 
```
make twoliter build kit bottlerocket-core-kit --arch x86_64 -e BUILDSYS_UPSTREAM_SOURCE_FALLBACK=true
```

Reply:
Hm, maybe my comment only applies to `make twoliter ...` workflows. e.g., I will use

```
BUILDSYS_ARCH=x86_64 PACKAGE=glibc make twoliter build-package
```

To build just one package for a specific arch.